### ISSUE TITLE
Multiple links to share webcam feed with different expiration dates.

### ISSUE THREAD

**Issue by kennethjiang** [kennethjiang/OctoPrint-Anywhere#42]

**Comment by cnlson** [comment]
i was just going to log a request for this when i had someone that i sent the link to a while ago tell me it still worked.

**Comment by kennethjiang** [comment]
Hi @cnlson  you should be able to remove the sharable link, as in the attached screenshot.
<img width="2105" alt="screen shot 2018-04-03 at 11 08 36 am" src="https://user-images.githubusercontent.com/779786/38267125-69b35cae-372f-11e8-8cd1-2a305a11bdf8.png">

Let me know if the link is still active even after you turn it off.

**Comment by cnlson** [comment]
Yes but I'd like to be able to share when I'm doing a project for someone and don't want to continue sharing months down the line. 

Maybe generate a new hash each time it is turned off and back on?

**Comment by kennethjiang** [comment]
Every time you turn the sharing off and turn it on again it'll be a different link. Let me know if it's not the case for you @cnlson

**Comment by cnlson** [comment]
you are correct. I don't know if i hadn't disabled it in a while or what. but thank you!
